FEM fails to compile under MSVC

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

When compiling from current main (26 March 2024), FEM fails to compile under MSVC, complaining:
```
src\Mod\Fem\App\FemPostObject.cpp(138): error C2872: 'vtkWriter': ambiguous symbol
LibPack-0.21-V2.10\include\vtk-8.2\vtkWriter.h(42): note: could be 'vtkWriter'
src\Mod\Fem\App\FemPostObject.cpp(78): note: or       'void `anonymous-namespace'::vtkWriter(const char *,const vtkSmartPointer<vtkDataObject> &)'
src\Mod\Fem\App\FemPostObject.cpp(138): error C2059: syntax error: '<'
```
The anonymous namespace can be resolved using `::vtkWriter`, but that still leaves me with a strange syntax error (that doesn't look like an error to me).

### Full version info

```shell
n/a can't compile
```


### Subproject(s) affected?

FEM

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/src/Mod/Fem/App/FemPostObject.cpp
+++ b/src/Mod/Fem/App/FemPostObject.cpp
@@ -75,7 +75,7 @@ namespace
 {
 
 template<typename T>
-void vtkWriter(const char* filename, const vtkSmartPointer<vtkDataObject>& dataObject)
+void femVTKWriter(const char* filename, const vtkSmartPointer<vtkDataObject>& dataObject)
 {
     if (vtkDataSet::SafeDownCast(dataObject)->GetNumberOfPoints() <= 0) {
         throw Base::ValueError("Empty data object");
@@ -135,5 +135,5 @@ void FemPostObject::writeVTK(const char* filename) const
         name = name.append(".").append(extension);
     }
 
-    vtkWriter<vtkXMLDataSetWriter>(name.c_str(), data);
+    femVTKWriter<vtkXMLDataSetWriter>(name.c_str(), data);
 }

